buildenv/jenkins/testJobTemplate

```diff
@@ -143,6 +143,9 @@ ARCH_OS_LIST.each { ARCH_OS ->
 							stringParam('OPENJ9_REPO', OPENJ9_REPO, "OpenJ9 git repo. Please use ssh for zos.")
 							stringParam('OPENJ9_BRANCH', "master", "OpenJ9 branch")
 							stringParam('OPENJ9_SHA', "", "OpenJ9 sha")
+							stringParam('JDK_REPO', "", "JDK git repo. Please use ssh for zos.")
+							stringParam('JDK_BRANCH', "", "JDK branch")
+							stringParam('JDK_SHA', "", "JDK sha")
```

Comment:
We have both `JDK_SHA` and `OPENJDK_SHA ` this template. What is the difference? I do not see `JDK_SHA` is used in build.xml.

Reply:
I was planning to rename OPENJDK_SHA to JDK_SHA originally. However I would prefer it to be done at next stage. For now it should not be added. Will remove it.